core/src/main/scala/kafka/coordinator/GroupMetadata.scala

```diff
@@ -282,6 +289,18 @@ private[coordinator] class GroupMetadata(val groupId: String, initialState: Grou
     pendingOffsetCommits ++= offsets
   }
 
+  def removeOffsets(tpList: Seq[TopicPartition]): mutable.HashMap[TopicPartition, OffsetAndMetadata] = {
```

Comment:
nit: instead of `tpList`, maybe something like `topicPartitions` or `partitions` is clearer? There are a couple other cases like this. Also I'm wondering if the return type needs to be mutable.